app/src/main/res/layout/item_tx_transfer.xml

```diff
@@ -47,6 +47,8 @@
         android:layout_marginStart="@dimen/default_small_margin"
         android:layout_marginEnd="@dimen/default_small_margin"
         android:gravity="end"
+        android:ellipsize="end"
+        android:maxLines="1"
```

Comment:
Would we need to give the same treatment to Custom transactions? As the method name could also be "unreasonably" long

Reply:
we do it already for custom tx